lib/logidze/has_logidze.rb

```diff
@@ -13,6 +13,10 @@ module ClassMethods # :nodoc:
       def has_logidze
         include Logidze::Model
       end
+
+      def has_logidze?
+        included_modules.include? Logidze::Model
```

Comment:
Yep, you're right. Than we can check for `has_logidze?` method itself too (i.e. `respond_to?(:has_logidze?)`. It's still a little bit faster than checking included modules.